client/layout/masterbar/notifications.jsx

```diff
@@ -23,32 +30,51 @@ export default React.createClass( {
 		tooltip: React.PropTypes.string,
 	},
 
+	componentDidMount() {
+		wpcom
```

Comment:
on the other hand, I don't think there would be any problem completely integrating this as a Calypso-only app. it isn't used outside of Calypso so we could more directly tie in `connect()` and friends inside the app